bin/install_travis.sh

```diff
@@ -86,7 +86,8 @@ if [[ "${WITH_MPC}" == "yes" ]]; then
 fi
 
 if [[ "${WITH_FLINT}" == "yes" ]]; then
-    conda_pkgs="$conda_pkgs libflint=2.5.2"
+    git clone https://github.com/wbhart/flint2;
+    cd flint2 && git checkout 06defcbc52efe41a8c73496ffde9fc66941e3f0d && ./configure --prefix=$our_install_dir && make -j8 install && cd ..;
```

Comment:
Also,  this [build](https://travis-ci.org/symengine/symengine/jobs/234756914) failed as we are trying here to configure `flint2` before `gmp` is installed. I think this `git clone...` should be done after all conda_pkgs are installed(Only If the flag `WITH_FLINT_DEV` is `yes`).